install/2_4_install/savu_v2.4/environment.yml

```diff
@@ -27,11 +28,12 @@ dependencies:
 - xraylib
 - tomopy
 - pyfai
+- fabio
+- sphinx-rtd-theme
+- gnureadline
+- pymca==5.4.3
```

Comment:
If you have fixed compatibility with the latest PyMca release, I think you cannot fix it to this older release.

Reply:
Thanks!  Just wanted to check the test passed on Travis before I removed it from the installation :-)